docs/pages/tutorials/username-and-password/nextjs-app.md

```diff
@@ -177,7 +177,7 @@ interface ActionResult {
 }
 ```
 
-In the form action, first do a very basic input validation. Get the user with the username and verify the password. If successful, create a new session with `Lucia.createSession()` and set a new session cookie.
+In the form action, First, do a very basic input validation. Get the user with the username and verify the password. If successful, create a new session with `Lucia.createSession()` and set a new session cookie.
```

Comment:
I think this can be reverted

Reply:
Oops, that was a find-and-replace error. Good catch. This change has been reverted.